src/test/java/it/gov/pagopa/pu/debtpositions/mapper/InstallmentPIIMapperTest.java

```diff
@@ -83,7 +82,7 @@ void testMapWithNoPIINotNull(){
 
   //region map(it.gov.pagopa.pu.debtpositions.model.InstallmentNoPII)
   @Test
-  void testMapInstallmentNoPII(){
+  void testMapInstallmentNoPII() {
     //given
     InstallmentNoPII installmentNoPII = buildInstallmentNoPII();
     InstallmentPIIDTO installmentPIIDTO = buildInstallmentPIIDTO();
```

Comment:
can you check if the syncStatus inner fields are not null (checkNotNullFields on the syncStatus object)?